README.md

```diff
@@ -17,7 +17,7 @@ Not Yet. Probably not ever :(
 ### Manual
 
 1. [Download the .zip](https://github.com/mrmartineau/SetiUI-Icons-Sublime/archive/master.zip).
-2. Unzip and Copy the folder into `Packages/User` directory, which you can find using the menu item `Preferences -> Browse Packages...` in Sublime Text.
+2. Unzip and Copy the folder into `Packages` directory, which you can find using the menu item `Preferences -> Browse Packages...` in Sublime Text.
 3. rename the unzipped folder to the exact name of your current theme's parent folder, for example "Theme - Soda"
 4. If your theme is Soda Dark 3, then create a new file called "Soda Dark 3" in the newly downloaded directory.
```

Comment:
This is what I did: https://github.com/swill/Theme-SolarizedSwill

This theme is very static to my use case, but you can get an idea from it.  I could not find a way to theme the drawer without overriding the `Default.sublime-theme` file, but that file is a global config file and will make the drawer themed with that theme regardless of which theme you select.  

Anyway, hope this is helpful for you...